src/main/java/eu/yals/ui/AppView.java

```diff
@@ -68,6 +67,6 @@ public AppView(GitService gitService, AppUtils appUtils) {
 
   @Override
   public void configurePage(InitialPageSettings settings) {
-    settings.addFavIcon("icon", "/images/logo.png", "512x512");
```

Comment:
Re-check
Different favicons for https://dev.yals.eu/ and https://dev.yals.eu/favicon.ico. 
Second is better